### DISCUSSION TITLE
Explicit configuration replacement

### DISCUSSION THREAD

**Discussion by Jaysok** [vladimir-bukhtoyarov/bucket4j#459] [Q&A] [answered]
According to [the documentation](https://bucket4j.com/8.9.0/toc.html#implicit-configuration-replacement), my understanding is that it's necessary to explicitly call `replaceConfiguration` in my case. However, I'm wondering whether `replaceConfiguration` needs to be invoked for every client sharing a bucket, will all clients automatically become aware of the configuration change after a single command by a checking the state of the bucket?

So, to clarify, is it necessary to invoke `replaceConfiguration` for each individual client sharing a bucket, or is a single invocation by one client sufficient to update the configuration for all?

**Comment by vladimir-bukhtoyarov** [comment, marked as answer]
Hello @Jaysok 

It is enough to call ```replaceConfiguration``` from one client independently of amount of clients that share same bucket.
